aws/apis/aws.go

```diff
@@ -14,26 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
```

Comment:
why this filename change? gcp to aws?

Reply:
Git internally does not really care about renames, it just tracks content. It uses a `similarity index` to guess if a file is a rename. This can be adjusted in a local git config if desired. I am guessing GitHub is detecting a <50% change in content here so is assuming it is a rename. More context [here](https://git-scm.com/docs/git-diff-index#Documentation/git-diff-index.txt---find-renamesltngt).

Reply:
as long as we are not mixing providers unintentionally! :)